extension/src/popup/views/__tests__/Swap.test.tsx

```diff
@@ -138,44 +173,96 @@ describe("Swap", () => {
       } as any),
     );
   });
-  beforeEach(async () => {
-    const history = createMemoryHistory();
-    history.push(ROUTES.swap);
-    mockHistoryGetter.mockReturnValue(history);
-
-    act(() => {
```

Comment:
unfortunately doing this render in the `beforeEach` made it impossible to overwrite `jest
      .spyOn(ApiInternal, "getAccountIndexerBalances")` with different mock balances in individual tests. `render` in each test is a little more verbose, but it's easier to change data for each test